src/modules/RangeTestModule.cpp

```diff
@@ -234,8 +235,8 @@ bool RangeTestModuleRadio::appendFile(const meshtastic_MeshPacket &mp)
         }
 
         // Print the CSV header
-        if (fileToWrite.println(
-                "time,from,sender name,sender lat,sender long,rx lat,rx long,rx elevation,rx snr,distance,hop limit,payload")) {
+        if (fileToWrite.println("time,from,sender name,sender lat,sender long,rx lat,rx long,rx elevation,rx rssi,rx "
```

Comment:
This is minor, but I feel like we should keep the ordering for existing fields in case any users are relying on that order being the same. Suggestion would be to put rx_rssi on the end as the last field.